{"pytest_version": "7.4.3", "$report_type": "SessionStart"}
{"nodeid": "tests38_30_5.py::test_largest_subset", "location": ["tests38_30_5.py", 18, "test_largest_subset"], "keywords": {"test_largest_subset": 1, "tests38_30_5.py": 1, "Folder_30": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00015883002197369933, "start": 1700526656.6482487, "stop": 1700526656.6484082, "$report_type": "TestReport", "item_index": 2, "worker_id": "gw1", "testrun_uid": "fc184113c073446eac6c088193f74616", "node": "<WorkerController gw1>"}
{"nodeid": "tests38_30_5.py::test_smaller_sets", "location": ["tests38_30_5.py", 6, "test_smaller_sets"], "keywords": {"test_smaller_sets": 1, "tests38_30_5.py": 1, "Folder_30": 1}, "outcome": "passed", "longrepr": null, "when": "setup", "user_properties": [], "sections": [], "duration": 0.00018286099657416344, "start": 1700526656.648319, "stop": 1700526656.6485028, "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "fc184113c073446eac6c088193f74616", "node": "<WorkerController gw0>"}
{"nodeid": "tests38_30_5.py::test_smaller_sets", "location": ["tests38_30_5.py", 6, "test_smaller_sets"], "keywords": {"test_smaller_sets": 1, "tests38_30_5.py": 1, "Folder_30": 1}, "outcome": "failed", "longrepr": {"reprcrash": {"path": "/data/Benchmark_GPT3_5/Q38/gpt_3_5_turbo_results_5/Folder_30/generated_answer.py", "lineno": 7, "message": "Failed: Timeout >7.0s"}, "reprtraceback": {"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_smaller_sets():", "        set_lengths = {i for i in range(1, 94)}", "        for i in set_lengths:", "            initial_set = {j for j in range(i)}", ">           assert find_subset_of_length_n(initial_set) == 0"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q38/gpt_3_5_turbo_results_5/Folder_30/tests38_30_5.py", "lineno": 11, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    subsets_of_length_n = [s for s in range(total_subsets) if bin(s).count('1') == 94]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q38/gpt_3_5_turbo_results_5/Folder_30/generated_answer.py", "lineno": 7, "message": "in find_subset_of_length_n"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   subsets_of_length_n = [s for s in range(total_subsets) if bin(s).count('1') == 94]", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [[".0", "<range_iterator object at 0x7f22cd12fdb0>"]]}, "reprlocals": null, "reprfileloc": {"path": "Q38/gpt_3_5_turbo_results_5/Folder_30/generated_answer.py", "lineno": 7, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, "sections": [], "chain": [[{"reprentries": [{"type": "ReprEntry", "data": {"lines": ["    def test_smaller_sets():", "        set_lengths = {i for i in range(1, 94)}", "        for i in set_lengths:", "            initial_set = {j for j in range(i)}", ">           assert find_subset_of_length_n(initial_set) == 0"], "reprfuncargs": {"args": []}, "reprlocals": null, "reprfileloc": {"path": "Q38/gpt_3_5_turbo_results_5/Folder_30/tests38_30_5.py", "lineno": 11, "message": ""}, "style": "long"}}, {"type": "ReprEntry", "data": {"lines": ["    subsets_of_length_n = [s for s in range(total_subsets) if bin(s).count('1') == 94]"], "reprfuncargs": null, "reprlocals": null, "reprfileloc": {"path": "Q38/gpt_3_5_turbo_results_5/Folder_30/generated_answer.py", "lineno": 7, "message": "in find_subset_of_length_n"}, "style": "short"}}, {"type": "ReprEntry", "data": {"lines": [">   subsets_of_length_n = [s for s in range(total_subsets) if bin(s).count('1') == 94]", "E   Failed: Timeout >7.0s"], "reprfuncargs": {"args": [[".0", "<range_iterator object at 0x7f22cd12fdb0>"]]}, "reprlocals": null, "reprfileloc": {"path": "Q38/gpt_3_5_turbo_results_5/Folder_30/generated_answer.py", "lineno": 7, "message": "Failed"}, "style": "long"}}], "extraline": null, "style": "long"}, {"path": "/data/Benchmark_GPT3_5/Q38/gpt_3_5_turbo_results_5/Folder_30/generated_answer.py", "lineno": 7, "message": "Failed: Timeout >7.0s"}, null]]}, "when": "call", "user_properties": [], "sections": [["Captured stderr call", "\n~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (139787465913920) ~~~~~~~~~~~~~~~~~~~~~\n  File \"/homes/sh2921/.local/lib/python3.10/site-packages/execnet/gateway_base.py\", line 361, in _perform_spawn\n    reply.run()\n  File \"/homes/sh2921/.local/lib/python3.10/site-packages/execnet/gateway_base.py\", line 296, in run\n    self._result = func(*args, **kwargs)\n  File \"/homes/sh2921/.local/lib/python3.10/site-packages/execnet/gateway_base.py\", line 1049, in _thread_receiver\n    msg = Message.from_io(io)\n  File \"/homes/sh2921/.local/lib/python3.10/site-packages/execnet/gateway_base.py\", line 507, in from_io\n    header = io.read(9)  # type 1, channel 4, payload 4\n  File \"/homes/sh2921/.local/lib/python3.10/site-packages/execnet/gateway_base.py\", line 474, in read\n    data = self._read(numbytes - len(buf))\n"]], "duration": 7.000709473970346, "start": 1700526656.6487954, "stop": 1700526663.649506, "extras": [], "$report_type": "TestReport", "item_index": 0, "worker_id": "gw0", "testrun_uid": "fc184113c073446eac6c088193f74616", "node": "<WorkerController gw0>"}
{"exitstatus": 2, "$report_type": "SessionFinish"}
